src/yuzu/configuration/config.cpp

```diff
@@ -1087,20 +1119,25 @@ void Config::SaveAudioValues() {
 void Config::SaveControlValues() {
     qt_config->beginGroup(QStringLiteral("Controls"));
 
+    Settings::values.players.SetGlobal(!IsCustomConfig());
     for (std::size_t p = 0; p < Settings::values.players.GetValue().size(); ++p) {
         SavePlayerValue(p);
     }
+    WriteGlobalSetting(Settings::values.use_docked_mode);
```

Comment:
```suggestion
    // Disable docked mode if handheld is selected
    const auto controller_type = Settings::values.players.GetValue()[0].controller_type;
    if (controller_type == Settings::ControllerType::Handheld) {
        Settings::values.use_docked_mode.SetGlobal(!IsCustomConfig());
        Settings::values.use_docked_mode.SetValue(false);
    } else {
        Settings::values.use_docked_mode.SetGlobal(true);
    }
    WriteGlobalSetting(Settings::values.use_docked_mode);
```
We can set the global state here when handheld is selected. To allow setting and restoring the config when the game is running or stopped.  

If any other profile is used other than handheld we should keep the global value. Otherwise we need to display this setting on the Qt interface to notify the user that it will be overridden by the custom config .